stm32cube/stm32f7xx/drivers/src/stm32f7xx_hal_mmc.c

```diff
@@ -1167,6 +1167,13 @@ HAL_StatusTypeDef HAL_MMC_ReadBlocks_DMA(MMC_HandleTypeDef *hmmc, uint8_t *pData
     hmmc->ErrorCode |= HAL_MMC_ERROR_PARAM;
     return HAL_ERROR;
   }
+  if ((uint32_t)pData & 3)
```

Comment:
While this is fine for debugging, I strongly believe it does not belong in production code. And we have never seen a case where it triggered. But `printk` is from zephyr, and this is NOT in zephyr!